prow/config/jobs/proxy.yaml

```diff
@@ -0,0 +1,43 @@
+org: istio
+repo: proxy
+support_release_branching: true
+image: gcr.io/istio-testing/prowbazel:0.5.16
+node_selector:
+  testing: build-pool
+branches: [prow-stage]
+
+jobs:
+- name: test
+  type: presubmit
+  repos: [istio/istio@master]
```

Comment:
Should these be istio/proxy?

Reply:
`repos` are extra repos to checkout, proxy is implicitly checked out at the PR commit for presubmit and the commited PR for for postsubmit.

The proxy jobs all explicitly configured this before, so I assumed they depend on something in istio/istio, is that not needed?

Reply:
I don't think it is necessary. Maybe @PiotrSikora would have better idea. I think it is fine to remove it.

Reply:
Well this is just for a dummy test branch anyways, so I'll just remove it then add it back later if we need it. No risk of breaking something

Reply:
sg.